### PR TITLE
Extensive admin

### PR DESCRIPTION
**Pull request by randall-sim**

### CODE REVIEW

**Comment by GandyT** [comment]
Corresponding backend pull request:
https://github.com/stuysu/api.stuyactivities.org/pull/762

File: src/comps/pages/admin/AdminMeetingCard.js [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+		fontWeight: 700
+	},
+	buttonContainer: {
+		width: "100%",
+		height: "50px",
+		bottom: 0
+	}
+};
+/* Meeting Card to let admin manage meeting */
+const AdminMeetingCard = ({
```
Comment (TheEgghead27) [inline src/comps/pages/admin/AdminMeetingCard.js:45]: Normally I would encourage reuse of the normal MeetingCard but hey if it works it works; will deploy to the beta domain for testing

**Review by TheEgghead27 — COMMENTED** [review]
Not approving for merge until https://github.com/stuysu/api.stuyactivities.org/pull/762 is done, and I would like to test the interface on beta branch first

File: src/pages/admin/manageclubs.js [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+	return (
+		<Box sx={classes.mainDiv}>
+			<OrganizationPicker setOrgId={setOrgId} />
+			<Grid container>
+				{data &&
+					data.organization &&
+					data.organization.meetings.map((meeting, i) => (
+						<Grid item xs={12} sm={12} md={6} lg={6} xl={4}>
+							<AdminMeetingCard
```
Comment (TheEgghead27) [inline src/pages/admin/manageclubs.js:83]: Can't seem to get this to show up on https://beta.stuyactivities.org/admin/manageclubs, even after adding a new meeting with a room set in the future.

**Review by TheEgghead27 — CHANGES_REQUESTED** [review]
Failed to test

**Review by TheEgghead27 — APPROVED** [review]

**Pull request merged**